src/main/webapp/openid-prototype.js

```diff
@@ -111,7 +111,7 @@ var openid = {
 	 * @return {Boolean}
 	 */
     submit : function() {
-      if (this.onsubmit())
+      //if (this.onsubmit())
```

Comment:
So in my attempts to figure out what was happening on the failed tests, this actually getting hit with an error message that the input item (the input of type submit) didn't have an onsubmit method.  It was strange, and I know the right object was being clicked from examination.  It was confusing, and a leftover from when I was trying to get the tests to work.